{"uuid":"e3eca565-adcf-4ebc-9219-9b6773772fae","historyId":"ee197713a25c7b8073d44ce1c94dcad6","fullName":"webUITest.TestObject.CreateProjectRoute2","labels":[{"name":"package","value":"webUITest.TestObject"},{"name":"testClass","value":"webUITest.TestObject"},{"name":"testMethod","value":"CreateProjectRoute2"},{"name":"parentSuite","value":"ParallelSuite"},{"name":"suite","value":"ParallelTest"},{"name":"subSuite","value":"webUITest.TestObject"},{"name":"host","value":"DESKTOP-KLI7F70"},{"name":"thread","value":"336@DESKTOP-KLI7F70.TestNG-test=ParallelTest-2(21)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"CreateProjectRoute2","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.hibernate.HibernateException: Flush during cascade is dangerous","trace":"javax.persistence.PersistenceException: org.hibernate.HibernateException: Flush during cascade is dangerous\r\n\tat org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:154)\r\n\tat org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:181)\r\n\tat org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:188)\r\n\tat org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1478)\r\n\tat org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:512)\r\n\tat org.hibernate.internal.SessionImpl.flushBeforeTransactionCompletion(SessionImpl.java:3310)\r\n\tat org.hibernate.internal.SessionImpl.beforeTransactionCompletion(SessionImpl.java:2506)\r\n\tat org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.beforeTransactionCompletion(JdbcCoordinatorImpl.java:447)\r\n\tat org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.beforeCompletionCallback(JdbcResourceLocalTransactionCoordinatorImpl.java:178)\r\n\tat org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.access$300(JdbcResourceLocalTransactionCoordinatorImpl.java:39)\r\n\tat org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:271)\r\n\tat org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:104)\r\n\tat hibernate.HibernateMain.closeSession(HibernateMain.java:39)\r\n\tat listeners.AllureTestListener.onTestFailure(AllureTestListener.java:35)\r\n\tat org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:99)\r\n\tat org.testng.internal.invokers.TestInvoker.runTestResultListener(TestInvoker.java:277)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:764)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\nCaused by: org.hibernate.HibernateException: Flush during cascade is dangerous\r\n\tat org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1467)\r\n\t... 22 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"browser","value":"chrome"}],"start":1685980140579,"stop":1685980140588}